src/prop.ts

```diff
@@ -201,14 +208,14 @@ function baseProp(rawOptions: any, Type: any, target: any, key: string, whatis:
   if (typeof ref === 'string') {
     schema[name][key] = {
       ...schema[name][key],
-      type: mongoose.Schema.Types.ObjectId,
+      type: rawOptions.refType || mongoose.Schema.Types.ObjectId,
```

Comment:
i think it would be better to do something like `rawOptions.refType = rawOptions.refType ? rawOptions.refType : mongoose.Schema.Types.ObjectId` instead of repetitively doing this